projects/storefrontapp-e2e-cypress/cypress/e2e/regression/cart/cart.e2e.cy.ts

```diff
@@ -47,6 +53,7 @@ describe('Cart', () => {
       });
 
       it('should keep cart on page refresh', () => {
+        cy.visit('/cart');
```

Comment:
just needed the navigation to /cart to be compatible with TestIsolation. so no need to flag the full 'describe' block